compiler/modules/bir/bir.bal

```diff
@@ -275,15 +345,15 @@ public type IntArithmeticBinaryInsn readonly & record {|
     *InsnBase;
     INSN_INT_ARITHMETIC_BINARY name = INSN_INT_ARITHMETIC_BINARY;
     ArithmeticBinaryOp op;
-    Register result;
+    TempRegister result;
```

Comment:
Create a (non-readonly) type that combines InsnBase with `TmpRegister result;` and `*` that (call it `ResultInsnBase`) instead of `InsnBase`.

Reply:
fixed